.helmdocsignore

```diff
@@ -2,3 +2,5 @@ charts/fluent-bit
 charts/grafana
 charts/loki
 charts/loki-stack
+charts/metrics-enterprise
+charts/promtail
```

Comment:
The promtail chart has been updated and should not be ignored

Reply:
Ah yeah apologies. I think that was a rebase artifact. I'll make sure to clean that up

Reply:
No problem, I just came across this and thought I'd drop a quick comment.